pynucastro/rates/approximate_rates.py

```diff
@@ -235,15 +290,38 @@ def __init__(self, primary_rate, secondary_rates,
         self._set_q()
 
     def get_child_rates(self):
-        """return a list of all of the rates that are used in this approximation"""
+        """Return a list of all of the rates that are used in this
+        approximation.
+
+        Returns
+        -------
+        list
+
+        """
         return list(self.rates.values())
 
     def _set_screening(self):
         # the individual rates are screened -- we don't screen the combination of them
         pass
 
     def eval(self, T, *, rho=None, comp=None):
-        """evaluate the approximate rate"""
+        """Evaluate the approximate rate.
+
+        Parameters
+        ----------
+        T : float
+            the temperature to evaluate the rate at
+        rho : float
+            the density to evaluate the rate at
+        comp : float
+            the composition (of type
+            :py:class:`Composition <pynucastro.networks.rate_collection.Composition>`)
+            to evaluate the rate with.
```

Comment:
Same for `ReaclibRate.eval()`.
```suggestion
        comp : Composition
            the composition to evaluate the rate with
```